packages/nx-plugin/src/executors/autorun/Readme.md

```diff
@@ -0,0 +1,56 @@
+# Autorun Executor
+
+#### @code-pushup/nx-plugin:autorun
+
+## Usage
+
+// project.json
+
+```json
+{
+  "name": "my-project",
+  "targets": {
+    "autorun": {
```

Comment:
I think most users will want the target to be `code-pushup`, so that they know what they're running:

```suggestion
    "code-pushup": {
```

Reply:
Thx! Was a "typo".